build/templates/examples.rst.mako

```diff
@@ -55,6 +55,6 @@ ${helper.get_rst_header_snippet(os.path.basename(e), '-')}
    :language: python
    :linenos:
    :encoding: utf8
-   :caption: `(${os.path.basename(e)}) <${example_url_base}/${e}>`_
+   :caption: `(${os.path.basename(e)}) <${example_url_base}/${e.replace('\\', '/')}>`_
```

Comment:
Please ensure you run the code-generator with this change. If any generated files (in `docs` and `generated` folder) have a diff, include them in the PR.

Reply:
please confirm and resolve

Reply:
There are no changes to generated files after building, except for line endings (CRLF vs LF).